chainList.json

```diff
@@ -1,7 +1,7 @@
 [
   {
-    "name": "OP-Mainnet",
-    "identifier": "mainnet/op",
+    "name": "OP Mainnet",
+    "identifier": "mainnet/oeth",
```

Comment:
What is the difference between `identifier` used here and `chainShortName` introduced by this PR? Could they be combined into a single field?

Reply:
The identifier includes the superchain target. The chain short name does not in general (sometimes it also does, but not in any standardized way). 

Having said that, I think this should revert to using the existing names [020a0ec](https://github.com/ethereum-optimism/superchain-registry/pull/337/commits/020a0ec9346b705d0e0d02fdf2575676ba369579)